lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/filteredStringListView.py

```diff
@@ -72,13 +72,11 @@ def __init__(self, data: StringListData, headerTitle: str = "", parent=None):
 
         self.setCursor(Qt.ArrowCursor)
 
-        DragAndDropEventFilter(self, data)
+        DragAndDropAndDeleteEventFilter(self, data)
 
         self.selectionModel().selectionChanged.connect(self.itemSelectionChanged)
 
     def paintEvent(self, event: QPaintEvent):
-        painter = QPainter(self)
-        painter.fillRect(event.rect(), Theme.instance().palette.colorListBorder)
```

Comment:
Not sure why we were painting this rectangle, I could not see a visual change when I removed it and it cased Qt to issue errors on stdout